packages/helpers/src/i18n-create-interpolate-element.js

```diff
@@ -0,0 +1,24 @@
+import { createInterpolateElement } from "@wordpress/element";
+import { sprintf } from "@wordpress/i18n";
+
+/**
+ * Create an interpolated element from a translated string and catches errors.
+ *
+ * @param {string} translatedString The translated string to be interpolated.
+ * @param {[string]} args The arguments to be interpolated into the translated string.
```

Comment:
`sprintf` does not need to be string only. You can use numbers and the like with `%d`.

Links:
- https://github.com/WordPress/gutenberg/blob/trunk/packages/i18n/src/sprintf.js
- https://github.com/alexei/sprintf.js